src/applications/pre-need/config/form.jsx

```diff
@@ -821,13 +836,7 @@ const formConfig = {
                   mailingAddress: merge(
                     {},
                     address.uiSchema('Mailing address'),
-                    {
-                      country: { 'ui:required': isAuthorizedAgent },
-                      street: { 'ui:required': isAuthorizedAgent },
-                      city: { 'ui:required': isAuthorizedAgent },
-                      state: { 'ui:required': isAuthorizedAgent },
-                      postalCode: { 'ui:required': isAuthorizedAgent },
-                    },
```

Comment:
Does this change in the uiSchema need to behind a flag?

Reply:
That's what it used to be, look at the right column for the updated change.